functional/tests/core-discovery-jsonrpc.test.mjs

```diff
@@ -0,0 +1,35 @@
+"use strict";
+
+import { expect, jest, test, afterAll, beforeAll } from "@jest/globals";
+import * as utilities from "./utilities.mjs";
+
+jest.setTimeout(30030);
+
+const httpPort = 3335;
+const socketPort = 9990;
+
+beforeAll(async () => {
+  console.log(
+    await utilities.mfState(
+      true,
+      ` -- --discovery --httpPort ${httpPort} --socketPort ${socketPort}`
+    )
+  );
+});
+
+afterAll(async () => {
+  //Stop Mock Firebolt
+  console.log(await utilities.killPort(socketPort));
+});
+
+test(`OPENRPC Response for discovery SDK`, async () => {
```

Comment:
can be changed to "Validate firebolt response for Discovery SDK"?

Reply:
Updated